core/scss/utilities/mixins/cta/_cta--button.scss

```diff
@@ -32,6 +25,14 @@
     }
   }
 
+  .su-cta__img {
+    @include image-defaults;
```

Comment:
All image tags get this by default. Do you need this again for a particular style?

Reply:
I didn't realize all image tags get this by default. Removed.